src/pages/Details.js

```diff
@@ -9,6 +10,17 @@ import Navigation from '../components/Navigation/Navigation';
 export default function Details({menusdata}) {
   const params = useParams();
   const currentMenu = menusdata?.find(menu => menu.id === params.id);
+  let actives = [false, false, false, false, false];
+
+  function setActive() {
+    for (let i = 0; i < currentMenu.spiciness; i++) {
+      if (currentMenu.spiciness < actives.length) {
+        actives[i] = true;
+      }
+    }
+  }
+  setActive();
```

Comment:
You can use this function:

```
function calculatePeppers(spicinessValue) {
  return [false, false, false, false, false].map(
    (_, index) => spicinessValue > index
  );
}
```

Reply:
Thank you, I never seen this before, looks cool!  I'll go find more info about it.